addons/api/addon/serializers/session-recording.js

```diff
@@ -11,4 +11,28 @@ export default class RecordingSerializer extends ApplicationSerializer.extend(
   attrs = {
     connection_recordings: { embedded: 'always' },
   };
+
+  serialize(snapshot) {
+    let serialized = super.serialize(...arguments);
+    // The post request for reapply-storage-policy doesn't expect a payload,
+    // so we just delete these
+    if (snapshot?.adapterOptions?.method === 'reapply-storage-policy') {
+      delete serialized.retain_until;
+      delete serialized.delete_after;
+      delete serialized.connection_recordings;
+    }
```

Comment:
Does the backend expect an `{}` or `null`? We should just return whichever they expect rather than deleting all fields that it doesn't.

Do they not need a version as part of the body?

Reply:
they don't need anything in the req body for the post call, that's why I deleted the fields

Reply:
Yeah I'd just return `{}` or `null` instead then rather than delete all the fields